src/authentication.ts

```diff
@@ -20,14 +20,14 @@ export function BasicAuth(app: Express.Application) {
   };
 
   app.use(passport.initialize());
-  app.use((req: Request, res, next) => {
+  app.use((req: CustomRequest, res, next) => {
     // If user already has user account details, it means it has already been authenticated
     // and we don't need to do it again. We rather skip to the next middleware.
-    if (req.session.user) {
-      req.user = req.session.user;
-      next();
-      return;
-    }
+    // if (req.session.user) {
```

Comment:
are we sure we want to comment this code?

Reply:
I'm working on it, but I'm fighting with cryptic ts errors :(

Reply:
No we don't want to, @Lqp1 is working on this

Reply:
Fixed!